apps/meeting/CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing Guidelines
+
+Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional
+documentation, we greatly value feedback and contributions from our community.
+
+Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
+information to effectively respond to your bug report or contribution.
+
+
+## Reporting Bugs/Feature Requests
+
+We welcome you to use the GitHub issue tracker to report bugs or suggest features.
+
+When filing an issue, please check existing open, or recently closed, issues to make sure somebody else hasn't already
+reported the issue. Please try to include as much information as you can. Details like these are incredibly useful:
+
+* A reproducible test case or series of steps
+* The version of our code being used
+* Any modifications you've made relevant to the bug
+* Anything unusual about your environment or deployment
+
+
+## Contributing via Pull Requests
+Contributions via pull requests are much appreciated. Before sending us a pull request, please ensure that:
+
+To send us a pull request, please:
+1. Fork the repository and make sure you are working against the latest source on the *meeting-dev* branch. We use *meeting-dev* branch for `apps/meeting` app to contribute all PRs. This app is used to test the changes releasing into the next [amazon-chime-sdk-component-library-react](https://github.com/aws/amazon-chime-sdk-component-library-react) release.
+2. Run the meeting demo locally first. The *meeting-dev* apps/meeting demo changes are unstable and contributors will need to create and install the `amazon-chime-sdk-component-library-react` tarball from its *master* branch to start the demo.
+
+    1. In a different directory, create `amazon-chime-sdk-component-library-react` tarball.
+    ```
+    git clone https://github.com/aws/amazon-chime-sdk-component-library-react &&
+    cd amazon-chime-sdk-component-library-react &&
```

Comment:
[for learning] we don't need to `npm run build` first?

Reply:
That is correct, I missed adding the install and build step, nice catch!